src/stories/RecapSection.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import RecapSection from "../components/RecapSection";
+
+export default {
+  title: "Recap Section",
+  component: RecapSection,
+};
+
+export const Default = {
+  args: {
+    videoLink:
+      "https://drive.google.com/file/d/19MXfja5TRSUH_1wXcsevFqXZPzKpEv7i/preview",
+    statistics: [
+      {
+        num: 4,
+        unit: "Days",
+      },
+      {
+        num: 13,
+        unit: "Speakers",
+      },
+      {
+        num: 588,
+        unit: "Participants",
+      },
+    ],
+    bgPath: "/recap-bg.png",
```

Comment:
could we import this with something like `import bgPath from "../assets/recap-bg.png"`, we're not publishing this component library as a site so it doesn't make sense to put our test assets in the `public` folder